packages/phone-number-privacy/common/src/utils/sequential-delay.ts

```diff
@@ -0,0 +1,80 @@
+import { SequentialDelayDomain, SequentialDelayStage } from '../interfaces'
+interface IndexedSequentialDelayStage extends SequentialDelayStage {
```

Comment:
Nit: A little whitespace would make this a bit easier to read 😅 
```suggestion

interface IndexedSequentialDelayStage extends SequentialDelayStage {
```